samples/unit-tests/series-column/gradient-in-format/demo.js

```diff
@@ -0,0 +1,35 @@
+QUnit.test("Preserve gradients in format strings(#2995)", function (assert) {
```

Comment:
Not really related to columns, move it to a new folder, `unit-tests/utilities/format/demo.js`.